### ISSUE TITLE
Create Templates for Enabler Feature and Business Feature

### ISSUE THREAD

**Issue by probbins79** [paladinarcher/accelerators#41]
**Description:** 
As an Architect, Product Manager, or a Product Owner, 
I want to have a simple Business Feature template and a simple Enabler Feature template that will include a link for further information and context, 
So that I can efficiently and quickly write a fill in the blank Business Feature or an Enabler Feature and have quick access to more detailed information in the link.
**Acceptance Criteria:** 
- [x] Given the current business feature and enabler template is too wordy when the new template is created it will delete non-essential information, then it will have the simple fill-in-the-blank ease of writing.
- [x] Given there are times when more context or information is needed when a business feature or enabler feature is being written, then there will be a link added to more context and information.
- [x] User Testing for This Story

https://paladinarcher.sharepoint.com/:w:/g/EcZeUGukawJKqcYMs5evUksB-N6pj3Gy6pl7e14uNBz_zg?e=Uc7nAX

**Comment by debsmith-ux** [comment]
Based on the diagram, it looks like whether a Feature is a Busines Feature or an Enabler Feature the template form is the sa
me...when they are broken into stories the template form changes to either User Story or Enabler Story with different templates. We will need to have a separate template for an Enabler Story.

**Comment by debsmith-ux** [comment]
Moroni  2 minutes ago
Business Feature and Enabler Feature needs different templates. A Business feature provides end user value - so persona's and user roles would be referenced.
An Enabler does not, and enabler feature does need to express the benefit to the business. Like what features does it support? Will it allow us to reduce risk, cut cost, increase value/quality/effectiveness, or make us more efficient?
Also sent to the channel

Moroni  1 minute ago
Thoughts on Business vs. Enabler features ^

Moroni  < 1 minute ago
And it would make sense to define multiple Enabler Feature templates: Design Enabler, Content Enabler, Technical Enabler.
